docs/troubleshoots.md

```diff
@@ -0,0 +1,30 @@
+# Troubleshoots 
+
+## Setting up Gmail to be used as default mail server in .env
+
+Gmail required applications that will use their service to be authorized first beforehand. Failure in doing this will cause SMTP sending mail process fail with the response **"534-5.7.9 Application-specific password required..."**.
+
+To solve this issue, you need to give this application an App password that is generated from Gmail account.
+
+Here are the steps:
+
+1. Login to your Gmail account and click on **Security** tab from the top navbar.
```

Comment:
Although , all the steps are correct but as @rpattath was saying about each and every step is crucial , no matter how small or basic it is , so i thought it would be great if you add one more line before this "Login" one, for like "Go to  https://myaccount.google.com/ " then login into your account and then click security [rest will be same as it is ]. Rest is all good and you have done a great job adding this @mtreacy002 :)

Reply:
Seems like a good first step! the URL to access is important :) @mtreacy002 could you do this as well please?

Reply:
After this little change I am happy to approve and merge :)

Reply:
got it. will work on it and push soon 😀

Reply:
done 😉

Reply:
Great 👍